scripts/code-rabbit-yaml.ts

```
@@ -0,0 +1,73 @@
+        .join('\n');
+}
+
+function appendInstructions(templateYaml: string, additions: Record<string, string>): string {
+    const doc = yaml.load(templateYaml) as {
+        reviews?: {
+            path_instructions?: Array<{
+                path: string;
+                instructions?: string;
+            }>;
+        };
+    };
```

ESLint

[error] Avoid casting objects. Add necessary type checks instead.

(@nangohq/custom-integrations-linting/no-object-casting)